hack/Dockerfile.unit

```diff
@@ -1,6 +1,10 @@
 FROM registry.hub.docker.com/library/golang:1.13
 
+COPY . /usr/local/kubebuilder/
 WORKDIR /usr/local/kubebuilder
-COPY /hack/tools /usr/local/kubebuilder/
+RUN go mod download -json
+
+WORKDIR /usr/local/kubebuilder/hack/tools
```

Comment:
you should set the workdir once only. for other needs, just use cd in your RUN commands

Reply:
okay